parent/pom.xml

```diff
@@ -114,6 +114,9 @@
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-archetype-plugin</artifactId>
                     <version>${maven.archetype.version}</version>
+                    <configuration>
+                        <ignoreEOLStyle>true</ignoreEOLStyle>
+                    </configuration>
```

Comment:
Not having this configuration gives a maven warning saying "Property ignoreEOLStyle was not set - files will be compared considering their EOL style!". 
This could cause issues in windows therefore I added it.